### PR TITLE
Update Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: po/extra/nl.po [hunk 1]
```diff
@@ -108,35 +104,35 @@ msgstr "Pictogram verbeterd (veel dank aan @Fatih20)"
...
+msgstr "Beschrijvingshulpballonnen toegevoegd"
 
 #: data/konbucase.appdata.xml.in:101
 msgid "Allow settin per-app dark style preference"
-msgstr ""
+msgstr "Ondersteuning voor donker thema per toepassing"
 
 #: data/konbucase.appdata.xml.in:109
 msgid "Add French translation (thanks to @NathanBnm)"
 msgstr "Franse vertaling (met dank aan Nathan Bonnemains)"
```
Comment (ryonakano) [inline po/extra/nl.po:131]:
```suggestion
msgstr "Franse vertaling (met dank aan @NathanBnm)"
```

This is not one of changes you made in this PR, but I wonder why the username in the source string is "translated" into the full name of him only here

**Review by ryonakano — CHANGES_REQUESTED** [review]
Hi @Vistaus, thanks again! Just one small thing and it's ready to go :rocket:

File: po/extra/nl.po (hunk 1 above)
Comment (Vistaus) [reply po/extra/nl.po]: You're right, that does look strange. I don't know how that happened. Thanks for noticing - it's fixed now! 🙂

File: po/extra/nl.po [hunk 2]
```diff
@@ -108,35 +104,35 @@ msgstr "Pictogram verbeterd (veel dank aan @Fatih20)"
...
 
 #: data/konbucase.appdata.xml.in:101
 msgid "Allow settin per-app dark style preference"
-msgstr ""
+msgstr "Ondersteuning voor donker thema per toepassing"
 
 #: data/konbucase.appdata.xml.in:109
 msgid "Add French translation (thanks to @NathanBnm)"
-msgstr "Franse vertaling (met dank aan Nathan Bonnemains)"
+msgstr "Franse vertaling (met dank aan <a class=
```
Comment (ryonakano) [inline po/extra/nl.po:131]:
```suggestion
msgstr "Franse vertaling (met dank aan @NathanBnm)"
```

Seems like you pasted wrong text  :sweat_smile:

**Review by ryonakano — CHANGES_REQUESTED** [review]

File: po/extra/nl.po (hunk 2 above)
Comment (Vistaus) [reply po/extra/nl.po:131]: I didn't paste anything - I clicked "Commit suggestion".

Comment (Vistaus) [reply po/extra/nl.po:131]: GitHub says that the original text is good now, so all should be good.

Comment (ryonakano) [reply po/extra/nl.po]: Hmm, not sure what happened here but now it is fixed.

**Review by ryonakano — APPROVED** [review]
Thank you again! :heart:

**Comment by Vistaus** [comment]
No problem, mate! 🙂

**Pull request merged**